resources/src/main.js

```diff
@@ -358,13 +358,35 @@ export function mountApp(rootSelector = '#labki-pack-manager-root') {
           return;
         }
         const payload = tree.exportSelectionSummary(this.activeRepo);
-        console.log('[Import payload]', payload);
+        // could instead send entire payload and have backend handle filtering. 
+        // Basically just tell backend what packs are selected and what action to take.
+        const selected = payload.packs.filter(p => p.selected && p.installStatus === 'new');
+        console.log('[Import payload]', selected);
 
-        // Example: send to backend
-        // const api = new mw.Api();
-        // await api.post({ action: 'labkiPackImport', format: 'json', payload: JSON.stringify(payload) });
+        this.pushMessage(MSG_TYPES.INFO, 'Starting import…');
 
-        this.pushMessage(MSG_TYPES.SUCCESS, 'Import triggered.');
+        try {
+          // maybe should put in api.js?
+          const api = new mw.Api();
```

Comment:
i am not seeing where `mw` gets defined? also can't find `postWithToken` method, is this some external package we're importing or where does this get defined